src/client/javascript/gui/containers.js

```diff
@@ -75,11 +75,14 @@
 
           var max = parseInt(cel.getAttribute('data-max-size'), 10);
           if ( !max ) {
-            var totalHeight = resizer.parentNode.offsetHeight;
+            var totalSize = resizer.parentNode.offsetHeight;
+            if (orient === 'horizontal') {
+              totalSize = resizer.parentNode.offsetWidth;
+            }
```

Comment:
If you could change it to `totalSize = resizer.parentNode[(orient === 'horizontal' ? 'offsetWidth' : 'offsetHeight')];` I'd be a happy man :)